ts/sveltelib/WithBase.svelte

```diff
@@ -0,0 +1,17 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script lang="typescript">
+    import useAsync from "sveltelib/async";
+    import { setupI18n } from "anki/i18n";
+    import { checkNightMode } from "anki/nightmode";
+
+    const nightMode = checkNightMode();
+
+    const { loading, value: i18n } = useAsync(() => setupI18n());
```

Comment:
Is this dead code? If not, setupI18n() call signature needs updating

Reply:
Oh, yes. I abandoned this, once I realized I had to do the i18n setup outside of Svelte.